pages/common/mh_metric.md

```diff
@@ -0,0 +1,24 @@
+# mh_metric
+
+> Calculate and enforce code metrics for MATLAB or Octave code.
+> More information: <https://misshit.org>.
+
+- Print the code metrics for the specified files:
+
+`mh_metric {{path/to/file1.m path/to/file2.m ...}}`
+
+- Print the code metrics for the specified Octave files:
+
+`mh_metric --octave {{path/to/file1.m path/to/file2.m ...}}`
+
+- Print the code metrics for the specified directory:
```

Comment:
Should we add `tree`. It seems from the documentation it does a recursive walk of the specified directory.
```suggestion
- Print the code metrics for the specified directory tree:
```

Reply:
@mfrw is `recursively` more appropriate?

Reply:
Yep I think that sounds much better.

Reply:
```suggestion
- Print the code metrics for the specified directory recursively:
```